File: Localidad.cc
#include "Localidad.h"

//PArte publica
Localidad::Localidad(string cadena){
	nombre=cadena;
 	id=-1;
 }

 Localidad::Localidad(const Localidad &l){
 	nombre=l.nombre;
 	coor=l.coor;
 	info=l.info;
 	id=l.id;
 }

 Localidad::~Localidad(){
 	nombre=0;
 	coor=0;
 	info=0;
 	id=0;
 }

Localidad::Localidad & operator=(const Localidad &l){
	if(this!=&l){
	 	nombre=l.nombre;
	 	coor=l.coor;
	 	info=l.info;
	 	id=l.id;
	 }
	 return *this;
}

int Localidad::setCoor(int ent1, int ent2, vector<vector<char>> mapa){
//FALTA HACER
}

void Localidad::setInfo(InfoTur inf){
	info=inf;
}

string Localidad::getNombre(){
	return nombre;
}

Coordenadas Localidad::getCoor(){
	return coor;
}

InfoTur Localidad::getInfo(){
	return info;
}

int Localidad::getId(){
	return id;
}

ostream & operator<<(ostream &os, const Localidad &l){
os c.id << "-" << c.nombre << "-" << c.coor.toString() << "\n" << c.info.toString() << endl;
return os;
}

File: Localidad.java
 import java.util.*

 public class Localidad{

 	private String nombre;
 	private Coordenadas coor;
 	private InfoTur info;
 	private int id;

 	public Localidad(String cadena){
 		nombre=cadena;
 		id=-1;
 	}

 	public int setCoor(int ent1, int ent2, ArrayList<ArrayList<Character>> mapa){

 		Coordenadas coord= new Coordenadas(ent1,ent2);
 		int n=0,id2=0 ;

 		n=mapa.get(0).size();
 		id2=n*(ent1+ent2);

 		if(coor==null || coor.get(ent1)==-1 || coor.get(ent2)==-1){
 			coor=coord;
 			if(mapa.get(ent1).get(ent2)=="T") {
 				mapa.get(ent1).get(ent2)=="L";
 				id=id2;
 				return id;
 			}
 			else return -1;
 		}

 	}

 	public void setInfo(InfoTur inf){
 		info=inf;
 	}
 	
 	public void setCoordenadas(Coordenadas coord) {
 		coor=coord;
 	}
 	
 	public void setId(int ide) {
 		id=ide;
 	}

 	public String getNombre(){
 		return nombre;
 	}

 	public Coordenadas getCoor(){
 		return coor;
 	}

 	public InfoTur getInfo(){ 
 		return info;
 	}

 	public int getId(){
 		return id;
 	}

 	public String toString(){
 		String dev= id + "-" + nombre + "-" + coor.toString() + "\n" + info.toString();
 		return dev;
 	}


 }